topics/proteomics/tutorials/clinical-mp-database-generation/tutorial.md

```diff
@@ -0,0 +1,326 @@
+---
+layout: tutorial_hands_on
+
+title: Clinical-MP-Database Generation
+zenodo_link: "https://doi.org/10.5281/zenodo.10105821"
+questions:
+- Why do we need to generate a customized database for metaproteomics research?
+- How do we reduce the size of the database?
+objectives:
+- Downloading databases related to 16SrRNA data
+- For better identification results, combine host and microbial proteins.
+- Reduced database provides better FDR stats.
+time_estimation: 3H
+key_points:
+- Create a customized proteomics database from 16SrRNA results.
+contributions:
+  authorship:
+    - subinamehta
+    - katherine-d21
+    - dechendb
+  editing:
+    - pratikdjagtap
+    - timothygriffin
+requirements:
+  -
+    type: "internal"
+    topic_name: proteomics
+subtopic: clinical-metaproteomics
+tags: [label-TMT11]
+---
+
+# Abstract
```

Comment:
please remove this section title, it will make it easier for us to generate previews of the tutorial (e.g. when shared on social media etc)